include/silo/common/aa_symbols.h

```diff
@@ -36,135 +39,33 @@ enum class AA_SYMBOL : char {
    X,     // Any amino acid
 };
 
-static constexpr uint32_t AA_SYMBOL_COUNT = 25;
-static constexpr uint32_t CODING_AA_SYMBOL_COUNT = 21;
+template <>
+class Util<AA_SYMBOL> {
+  public:
+   static constexpr uint32_t count = static_cast<uint32_t>(AA_SYMBOL::X) + 1;
 
-static constexpr std::array<AA_SYMBOL, AA_SYMBOL_COUNT> AA_SYMBOLS{
-   AA_SYMBOL::GAP, AA_SYMBOL::A, AA_SYMBOL::C, AA_SYMBOL::D, AA_SYMBOL::E,
-   AA_SYMBOL::F,   AA_SYMBOL::G, AA_SYMBOL::H, AA_SYMBOL::I, AA_SYMBOL::K,
-   AA_SYMBOL::L,   AA_SYMBOL::M, AA_SYMBOL::N, AA_SYMBOL::P, AA_SYMBOL::Q,
-   AA_SYMBOL::R,   AA_SYMBOL::S, AA_SYMBOL::T, AA_SYMBOL::V, AA_SYMBOL::W,
-   AA_SYMBOL::Y,   AA_SYMBOL::B, AA_SYMBOL::Z, AA_SYMBOL::X, AA_SYMBOL::STOP,
-};
+   static constexpr std::string_view symbol_name = "Amino Acid";
+   static constexpr std::string_view symbol_name_lower_case = "amino acid";
+   static constexpr std::string_view symbol_name_upper_case = "AMINO ACID";
+   static constexpr std::string_view symbol_name_short = "AA";
```

Comment:
Please check case style of constexpr values in this file. They should be UPPER_CASE.